screens/RoomListViewScreen/styles.js

```diff
@@ -0,0 +1,33 @@
+import { StyleSheet } from 'react-native'
+import EStyleSheet from 'react-native-extended-stylesheet';
+import { Colors } from '../../utils/colors';
+import { Typography } from '../../utils/typography';
+
+const styles = EStyleSheet.create({
+    img: {
+        backgroundColor: Colors.gray4,
+        flex: 3
+    },
+    roomname: {
+        ...Typography.subtitle1()
+    },
+    description: {
+        ...Typography.extraSmall()
+    },
+    caption: {
+        flex: 1,
+        padding: 10,
```

Comment:
do check ang ubang implementation do ga rem nami gi separate file napd namong styles

Reply:
To do, wala pa gi EStylesheet kay way intellisense